.github/workflows/deno_test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           deno-version: v1.x
```

Comment:
I can't comment on it but can we move the `transcationsData.json` file to `indexer/src/test-data`?